scripts/serve.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/zsh
```

Comment:
I'm no expert on zsh, so my comments below may be totally incorrect. Google style guide requires the use of bash, but I don't know if that applies here or not.